docs/user/supported_devices.rst

```diff
@@ -254,6 +254,9 @@ lantiq-xrx200
   - FRITZ!Box 7362 SL [#eva_ramboot]_ [#lan_as_wan]_
   - FRITZ!Box 7412 [#eva_ramboot]_
 
+* TP-Link
+  - TD-W8970 (v1)
```

Comment:
```suggestion
  - TD-W8970 (v1) [#lan_as_wan]_
```

The device has no dedicated wan port, but uses all ports for it, doesn't it?

Reply:
Yes, that's correct